src/cryptoadvance/specter/util/common.py

```diff
@@ -43,7 +46,7 @@ def btcamount_formatted(
     minimum_digits_to_strip=6,
     enable_digit_formatting=True,
 ):
-    value = round(float(value), 8)
+    value = Decimal(value).quantize(Decimal("0.00000001"))
```

Comment:
Why not use getcontext().prec = 8? If it does the same, I think it is more readable.

Reply:
This will infuriate and blow your mind. First, predict the outcome of the two print statements. Then run it.
```
import decimal
from decimal import Decimal

decimal.getcontext().prec = 8
d = Decimal("1234.1234567890")
print(d)
print(d + Decimal(0))
```

Reply:
Oh man, why are you doing this to me :-)